src/drawer.canvasentry.js

```diff
@@ -128,32 +132,35 @@ export default class CanvasEntry {
         this.waveCtx.clearRect(
             0,
             0,
-            this.waveCtx.canvas.width,
-            this.waveCtx.canvas.height
+            this.waveCtx.canvas[this.orientation.attrFor('width')],
+            this.waveCtx.canvas[this.orientation.attrFor('height')]
         );
 
         // progress
         if (this.hasProgressCanvas) {
             this.progressCtx.clearRect(
                 0,
                 0,
-                this.progressCtx.canvas.width,
-                this.progressCtx.canvas.height
+                this.progressCtx.canvas[this.orientation.attrFor('width')],
+                this.progressCtx.canvas[this.orientation.attrFor('height')]
             );
         }
     }
 
     /**
-     * Set the fill styles for wave and progress
+     * Set the drawing context, including fill styles and transform,
+     * for wave and progress
      *
      * @param {string} waveColor Fill color for the wave canvas
      * @param {?string} progressColor Fill color for the progress canvas
      */
-    setFillStyles(waveColor, progressColor) {
+    setDrawingContext(waveColor, progressColor) {
```

Comment:
This breaks the API. I'd rather keep the old setFillStyles around and introduce a new setDrawingContext-type method.